requirements.txt

```diff
@@ -1,5 +1,24 @@
-Flask
-redis
-rq>=0.3.8
-simplejson
-times
+arrow==0.5.4
+check-manifest==0.25
+click==4.0
+ecdsa==0.13
+Fabric==1.10.1
+flake8==2.4.1
+Flask==0.10.1
+grin==1.2.1
+itsdangerous==0.24
+Jinja2==2.7.3
+MarkupSafe==0.23
+mccabe==0.3
+paramiko==1.15.2
+pep8==1.5.7
+pycrypto==2.6.1
+pyflakes==0.8.1
+python-dateutil==2.4.2
+redis==2.10.3
+rq==0.5.3
+simplejson==3.7.3
+six==1.9.0
+times==0.7
+Werkzeug==0.10.4
+wheel==0.24.0
```

Comment:
Let's use [pip-tools](https://github.com/nvie/pip-tools/tree/future#installation) (the `future`) branch to compile the resolved dependencies, and only keep a lightweight top-level requirements.in file. This saves a ton of time doing package management. I'll add this to `master` after merging this PR.